src/blocklyc.js

```diff
@@ -220,24 +220,12 @@ var graph_data = {
 };
 
 
-/**
```

Comment:
This is no longer needed - the minimum versions use base64 encoding for serial streams